src/pages/SwapV2/index.tsx

```diff
@@ -529,7 +527,7 @@ export default function Swap({ history }: RouteComponentProps) {
     findTokenPairFromUrl()
     refIsImportUserToken.current = false
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [JSON.stringify(defaultTokens), refIsCheckNetworkAutoSelect.current])
+  }, [defaultTokens, refIsCheckNetworkAutoSelect.current])
```

Comment:
Please add a comment here why you need `defaultTokens` here though it's not in use

Reply:
used in `findTokenPairFromUrl` function

Reply:
Ok, I found it not reasonable, but we can skip for now. We'll do refactoring on this later. Any comment @viet-nv ?

Reply:
if defaultTokens  is used in `findTokenPairFromUrl` => findTokenPairFromUrl should be wrapped with useCallback, and defaultTokens is a dependency of that useCallback

Reply:
agree that you can skip it for now but please take a note about this

Reply:
<img width="756" alt="image" src="https://user-images.githubusercontent.com/33005392/178438291-cb3eab90-1fe4-41c9-8140-0127059d8365.png">
I only want to check the length of defaultTokens and return if length is zero. it's okay. I updated it: bring condition to here. is it okay?
<img width="792" alt="image" src="https://user-images.githubusercontent.com/33005392/178439839-d9be6f02-bc65-42c7-8196-ce1facbb4030.png">